README.md

```diff
@@ -60,4 +57,70 @@ These tables are an addition to the existing PEtab format to describe the covari
 
 - No table
 
-  If no table is provided, only the variances, but no covariances are estimated.
\ No newline at end of file
+  If no table is provided, only the variances, but no covariances are estimated.
+  
+## Measurement table
+
+In addition to the existing descriptions of fixed effects, the NLME extension adds the following columns:
+
+- `$covariateId` one column per distinct covariate
+  
+## Covariate table
```

Comment:
From the pharmacometrics perspective:
Usually different information from the patients is available (e.g. demographics, body weight, biochemistry markers, ...) that can be potential covariates in the model. 

During model development then the inclusion of the different covariates needs to be tested (as a model selection problem). Even the mathematical expression for the covariate may be tested to identify the most suitable one. That is why I kept it out from the SBML model, so the user will have high flexibility when defining the covariates and how they are modeled without needing to work with the SBML every time.

I see the point of adding the formula directly inside the SBML, but I thought this may be more complicated from a user perspective when iterating the structure.

So, I think maybe this depend on the approach we want to take:
(1) use PEtab while doing model development, or
(2) use PEtab to store the final model structure.

Reply:
OK, thanks for the explanation. I see why this makes the whole thing more user friendly. Here are some additional thoughts:

> the inclusion of the different covariates needs to be tested (as a model selection problem).

Then we need to make sure it is compatible with PEtab select. But my (limited) understanding of PEtab select suggests, that whatever we come up with here, can be nested inside a PEtab select problem, so we don't have to worry about this now.

Additionally, I am wondering if the `estimate` column in the covariate table should be replaced with sth like `select` (I think the `estimate` column should NOT be used to indicate whether $\theta_{cov}$, should be estimated or not. As I explain in a comment below, I think all $\theta_{cov}$ s should go explicitly in the `covariateFormula` and the parameter table (which has an `estimate` column already. What are your thoughts?).

The allowed values of such a `select` column would be sth like `include`, `exclude`, `optimize`. But what would you then do, if you have like three different formulas you want to try for sth. like `WT`? Well, I guess you could just copy the row for `WT`, except for the covariateFormula, which would be different. If `select` is `optimize` and no `WT` row is `include`, then this would mean to optimize which expression should be taken, if any (e.g. `WT` might not improve the model selection criterion, so no expression might be taken). If all rows are `exclude`, it is excluded, if exactly one row is `include`, it is included. Else the PEtab is invalid.

> I see the point of adding the formula directly inside the SBML, but I thought this may be more complicated from a user perspective when iterating the structure.

Fair enough. If for whatever reason this is needed (e.g. users prefer MathML over string expressions), users can still specify alternative SBML files as part of a stardard PEtab select problem I think.

> So, I think maybe this depend on the approach we want to take:

I am in favor of (1). PEtab is for model fitting/development, SBML for storing the final model with optimized parameters.